templates/pioneer-study/esper-study-telemetry.1.parquetmr.txt

```diff
@@ -0,0 +1,42 @@
+message esper-study-telemetry {
+  required group metadata {
```

Comment:
Just wanted to pop in here and say we have some expanded docs on the `metadata` section, check here [0]. Basically: if you think you might ever need it, add it, because you won't be able to later; available fields are given in a linked doc there as well.

[0] https://github.com/fbertsch/firefox-data-docs/blob/ee39bb8b1f3576236f27d27711ee0b0af29403a9/cookbooks/new_ping.md#submit-schema-to-mozilla-servicesmozilla-pipeline-schemas

Reply:
@fbertsch Thanks, this is important. Is this to say that if we are planning to use any of the application and environment data during data analysis, we will need to include it now into the metadata section later? Or are those automatically available?

Reply:
@motin application and environment data are available. The `metadata` are parts that are added server-side: hence `geoCountry`, `documentId`, etc. [Check here](https://hsadmin.trink.com/dashboard_output/analysis.trink.telemetry_schema.parquet.txt) for all available fields for each ping.